src/__tests__/snapshot-tests/__snapshots__/documenter.test.ts.snap

```diff
@@ -5982,51 +5982,10 @@ For more info on custom languages, see the [Code editor API](/components/code-ed
           ""d"",
           ""r"",
           ""properties"",
-          ""abc"",
```

Comment:
@just-boris Any idea what this stuff is? The documenter kicked it out after I moved the style prop to ButtonProps.

Reply:
I upgraded dependencies today: https://github.com/cloudscape-design/components/pull/3610

You need to run `npm install && npm run build` and then update the snapshots. They will be back to expected

Reply:
Okay!